src/scripts/breeding/Breeding.ts

```diff
@@ -53,6 +56,7 @@ class Breeding implements Feature {
             ['Dratini', 'Dragonair', 'Dragonite'],
             [],
             ['Bagon', 'Shelgon', 'Salamence'],
+            [],
```

Comment:
Will also need to add empty arrays to the rest of the Eggs that have no Pokemon

Reply:
Done